TTS/utils/audio.py

```diff
@@ -429,3 +428,13 @@ def quantize(x, bits):
     @staticmethod
     def dequantize(x, bits):
         return 2 * x / (2 ** bits - 1) - 1
+
+def log_(x, base):
```

Comment:
you can also raise an error if the base is not 10, or use the log with base in this case.

Reply:
> you can also raise an error if the base is not 10, or use the log with base in this case.

base only exist for known version of log_func and is either equal to np.e or 10, if it's not the case, an error is raised during init. Do you still want me to raise an error here ?
``` python
if log_func == "np.log":
    self.base = np.e
elif log_func == "np.log10":
    self.base = 10
else:
    raise ValueError(" [!] unknown `log_func` value.")
```

Reply:
> btw is this a naming convention that I don't know `name_ `?

No I didn't realy know how to call the function since it's not really log or exp... Suggestions ?

Reply:
fix looks cool! 

And ```_log ``` I think more convenient python naming for private class functions (out of my limited engineering nomenclature)

Reply:
I'm not sure I understand what you want me to do, replace log_ by _log ? At first I didn't want to do this since it's not in the class so it can't be a true a private function. But I really don't know so if you want me to name it this way, np !

Reply:
yep I meant just the renaming.